packages/@aws-cdk/aws-appmesh/lib/mesh.ts

```diff
@@ -99,6 +110,32 @@ abstract class MeshBase extends cdk.Resource implements IMesh {
       mesh: this,
     });
   }
+
+  public grantRead(identity: iam.IGrantable): iam.Grant {
+    return this.grant(identity,
+      'appmesh:DescribeMesh',
+      'appmesh:ListMeshes',
+      'appmesh:ListTagsForResource',
+    );
+  }
+
+  public grantWrite(identity: iam.IGrantable): iam.Grant {
+    return this.grant(identity,
+      'appmesh:CreateMesh',
+      'appmesh:UpdateMesh',
+      'appmesh:DeleteMesh',
+      'appmesh:TagResource',
+      'appmesh:UntagResource',
```

Comment:
Same questions here as for `GatewayRoute` - what would necessitate a resource having these permissions?